internal/view/helm_history.go

```diff
@@ -97,6 +103,43 @@ func (h *History) rollbackCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+func (h *History) yamlCmd(evt *tcell.EventKey) *tcell.EventKey {
```

Comment:
Unless I am missing it if memory servers yaml and describe we should get for free by virtue of the dao.Describer??

Reply:
Indeed, in the `Browser` if the resource category does not contains `k9sCat` then the yaml and describe commands are automatically added. So I changed the category of the helm-history resource to `helmCat` and it worked.